apps/src/templates/manageStudents/PasswordReset.jsx

```diff
@@ -59,6 +64,19 @@ class PasswordReset extends Component {
           isResetting: false,
           input: ''
         });
+        firehoseClient.putRecord(
```

Comment:
do we need to be able to distinguish records for this event from the reset-secret event in https://github.com/code-dot-org/code-dot-org/pull/34647?